### PR TITLE
Create Web API: HTTPS profile instructions

### LINKED ISSUE #28715: Default profile is http in dotnet 7.0.202, so no HTTPS instructions are applicable
Default configuration on dotnet 7.0.202 (Ubuntu installation) appears to create a Properties/launchSettings.json with the http profile before the https profile, making it the default.  As a result, none of the directions in the page are applicable, as the application does not start up with HTTPS enabled.

Include instructions on how to arrange the launchSettings.json profiles so that the https profile comes first, or otherwise select the profile when launching (which I have not figured out how to do yet).

---
#### Document Details

⚠ *Do not edit this section. It is required for learn.microsoft.com ➟ GitHub issue linking.*

* ID: 583ae3e7-5915-f22b-e54c-6c100cf2a7da
* Version Independent ID: 95f2549e-29f9-e31a-b4d0-432b971643ff
* Content: [Tutorial: Create a web API with ASP.NET Core](https://learn.microsoft.com/en-us/aspnet/core/tutorials/first-web-api?view=aspnetcore-7.0&tabs=visual-studio)
* Content Source: [aspnetcore/tutorials/first-web-api.md](https://github.com/dotnet/AspNetCore.Docs/blob/main/aspnetcore/tutorials/first-web-api.md)
* Product: **aspnet-core**
* Technology: **aspnetcore-tutorials**
* GitHub Login: @wadepickett
* Microsoft Alias: **wpickett**

---
[Associated WorkItem - 97034](https://dev.azure.com/msft-skilling/Content/_workitems/edit/97034)

### PR DESCRIPTION
**Pull request by wadepickett**
Fixes #28715

Changed the instruction on VS Code to specify the HTTPS profile for `dotnet run` on the command line.

---

#### Internal previews

| 📄 File | 🔗 Preview link |
|:--|:--|
| [aspnetcore/tutorials/first-web-api.md](https://github.com/dotnet/AspNetCore.Docs/blob/5aa54dea1716eaeac547e1b93d05b61a7eda6609/aspnetcore/tutorials/first-web-api.md) | [Tutorial: Create a web API with ASP.NET Core](https://review.learn.microsoft.com/en-us/aspnet/core/tutorials/first-web-api?branch=pr-en-us-29952) |

### CODE REVIEW

**Comment by wadepickett** [comment]
Closed and reopened to force new build.  Build was failing even though the edits in this PR were fine in last commit.  Forcing a new build did the trick.

File: aspnetcore/tutorials/first-web-api/includes/first-web-api3-7.md [hunk 1]
```diff
@@ -117,8 +117,26 @@ Visual Studio launches the default browser and navigates to `https://localhost:<
...
-* Visual Studio Code launches the default browser to `https://localhost:<port>`, where `<port>` is the randomly chosen port number displayed in the output. There is no endpoint at `https://localhost:<port>` so the browser returns [HTTP 404 Not Found](https://developer.mozilla.org/docs/Web/HTTP/Status/404). Append `/swagger` to the URL, `https://localhost:<port>/swagger`.
+* Run the following command to start the app on the `https` profile:
+
+  ```dotnetcli
+  dotnet run --launch-profile https
+  ```
+
+ The output shows messages similar to the following, indicating that the app is running and awaiting requests:
+
+   ```bash
```
Comment (tdykstra) [inline aspnetcore/tutorials/first-web-api/includes/first-web-api3-7.md:128]:
~~~suggestion
   ```output
~~~

File: aspnetcore/tutorials/first-web-api.md [hunk 2]
```diff
@@ -148,8 +148,26 @@ Visual Studio launches the default browser and navigates to `https://localhost:<
...
+   ```bash
+   ...
+   info: Microsoft.Hosting.Lifetime[14]
+         Now listening on: https://localhost:{port}
+   ...
+   ```
+
+* <kbd>Ctrl</kbd>+*click* the HTTPS URL in the output to test the web app in a browser.
+
+* The default browser is launched to `https://localhost:<port>`, where `<port>` is the randomly chosen port number displayed in the output. There is no endpoint at `https://localhost:<port>` so the browser returns [HTTP 404 Not Found](https://developer.mozilla.org/docs/Web/HTTP/Status/404). Append `/swagger` to the URL, `https://localhost:<port>/swagger`.
```
Comment (tdykstra) [inline aspnetcore/tutorials/first-web-api.md:168]:
```suggestion
* The default browser is launched to `https://localhost:<port>`, where `<port>` is the randomly chosen port number displayed in the output. There is no endpoint at `https://localhost:<port>`, so the browser returns [HTTP 404 Not Found](https://developer.mozilla.org/docs/Web/HTTP/Status/404). Append `/swagger` to the URL, `https://localhost:<port>/swagger`.
```

File: aspnetcore/tutorials/first-web-api/includes/first-web-api3-7.md [hunk 3]
```diff
@@ -117,8 +117,26 @@ Visual Studio launches the default browser and navigates to `https://localhost:<
...
+   ...
+   info: Microsoft.Hosting.Lifetime[14]
+         Now listening on: https://localhost:{port}
+   ...
+   ```
+
+* <kbd>Ctrl</kbd>+*click* the HTTPS URL in the output to test the web app in a browser.
+
+* The default browser is launched to `https://localhost:<port>`, where `<port>` is the randomly chosen port number displayed in the output. There is no endpoint at `https://localhost:<port>` so the browser returns [HTTP 404 Not Found](https://developer.mozilla.org/docs/Web/HTTP/Status/404). Append `/swagger` to the URL, `https://localhost:<port>/swagger`.
```
Comment (tdykstra) [inline aspnetcore/tutorials/first-web-api/includes/first-web-api3-7.md:137]:
```suggestion
* The default browser is launched to `https://localhost:<port>`, where `<port>` is the randomly chosen port number displayed in the output. There is no endpoint at `https://localhost:<port>`, so the browser returns [HTTP 404 Not Found](https://developer.mozilla.org/docs/Web/HTTP/Status/404). Append `/swagger` to the URL, `https://localhost:<port>/swagger`.
```

**Review by tdykstra — APPROVED** [review]
Looks good aside from a few nit suggestions.

**Pull request merged**